sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -159,6 +161,22 @@ object RapidsPluginUtils extends Logging {
         logWarning(s"$numThreadsKey is set to $numThreads.")
       }
     }
+
+    // If spark.task.resource.gpu.amount is larger than 
+    // (spark.executor.resource.gpu.amount / spark.executor.cores) then GPUs will be the limiting
+    // resource for task scheduling.
+    if (conf.contains(TASK_GPU_AMOUNT_KEY) && conf.contains(EXECUTOR_CORES_KEY) && 
+        conf.contains(EXECUTOR_GPU_AMOUNT_KEY)) {
```

Comment:
is spark.executor.cores set in standalone mode?  the default is use all cores when its not specified and I didn't think it got set.  I was going to ask local mode but gpu scheduling isn't supported there so doesn't matter.

Reply:
Yes, it is not set in standalone mode by default. I updated code to use worker's core num as default value. Thanks!